api/PayrollProcessor.Web.Api/Features/Employees/EmployeeUpdate.cs

```diff
@@ -33,23 +33,20 @@ public EmployeeUpdate(IQueryDispatcher queryDispatcher, ICommandDispatcher comma
             Tags = new[] { "Employees" })
         ]
         public override Task<ActionResult<Employee>> HandleAsync(EmployeeUpdateRequest request, CancellationToken token) =>
-            queryDispatcher.Dispatch(new EmployeeQuery(request.Id), token)
-                .Bind(employee =>
-                {
-                    var command = new EmployeeUpdateCommand(
-                        request.Email,
-                        request.EmploymentStartedOn,
-                        request.FirstName,
-                        request.LastName,
-                        request.Phone,
-                        request.Status,
-                        request.Title,
-                        request.Version,
-                        employee
-                    );
-
-                    return commandDispatcher.Dispatch(command);
-                })
+            queryDispatcher
+                .Dispatch(new EmployeeQuery(request.Id), token)
+                .Map(employee => new EmployeeUpdateCommand(
```

Comment:
Convert this to an expression to get rid of the statement/assignment.

Here's an example of where a static factory method on `EmployeeUpdateCommand` would be nice because we could call it via `.Map(EmployeeUpdateCommand.Create)` ... however, it would have to take `Employee` as a parameter, not the individual primitives.

This might be an ok approach since they are both `Core.Domain` types.